packages/loopring_v3/contracts/amm/libamm/AmmWithdrawProcess.sol

```diff
@@ -27,11 +27,14 @@ library AmmWithdrawProcess
     {
         require(ctx.settings.assetManager != IAssetManager(0), "CANNOT_WITHDRAW_FROM_POOL");
         require(poolWithdrawal.amounts.length == ctx.tokens.length, "INVALID_WITHDRAWAL_AMOUNTS");
+        // require(!S.exitMode, "WITHDRAWAL_DISALLOWED_IN_EXIT_MODE")
+
         for (uint i = 0; i < ctx.tokens.length; i++) {
             uint96 amount = poolWithdrawal.amounts[i];
             if (amount > 0) {
                 address token = ctx.tokens[i].addr;
                 verifyWithdrawalTx(ctx, ctx.tokens[i].tokenID, amount);
+                // token.transferOut(..., ...) where is the trasfer out?
```

Comment:
?

Reply:
This code is to authorize the withdrawal from the L2 account owned by the pool, so the actual token transfer will be done on the deposit contract.

Reply:
But where did we call the deposit contract to transfers the fund out to the AMM pool contract, I didn't find the code.

Reply:
That's done by the withdrawal transaction in the exchange contract like any other withdrawal.